ocs_ci/templates/cnv-vm-workload/vm-standalone-pvc/vm.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: centos-stream8
+  namespace: default
+spec:
+  running: true
+  template:
+    metadata:
+      annotations:
+        vm.kubevirt.io/flavor: small
+        vm.kubevirt.io/os: centos-stream8
+        vm.kubevirt.io/workload: server
+      creationTimestamp: null
+      labels:
+        kubevirt.io/domain: centos-stream8
+        kubevirt.io/size: small
+    spec:
+      architecture: amd64
+      domain:
+        cpu:
+          cores: 1
+          sockets: 1
+          threads: 1
+        devices:
+          disks:
+            - bootOrder: 1
+              disk:
+                bus: virtio
+              name: rootdisk
+            - bootOrder: 2
+              disk:
+                bus: virtio
+              name: cloudinitdisk
+          interfaces:
+            - name: default
+              masquerade: {}
+              model: virtio
+          networkInterfaceMultiqueue: true
+          rng: {}
+        machine:
+          type: pc-q35-rhel9.2.0
+        memory:
+          guest: 2Gi
+        resources: {}
+      evictionStrategy: LiveMigrate
+      networks:
+        - name: default
+          pod: {}
+      terminationGracePeriodSeconds: 180
+      volumes:
+        -  name: rootdisk
+           persistentVolumeClaim:
+             claimName: sample-vm-pvc
+        - cloudInitNoCloud:
+            userData: |-
+              #cloud-config
+              user: centos
+              password: 7lua-f8o8-nk3t
```

Comment:
Exposing anytype of plaintext username/passwords is not a good practice.

Reply:
Only the user who secret is added to the VM during the VM creation and has perms to the openshift console can access this VM. But, Yes I agree with you that this way is not a good practice. This is one of the way to send the userdata using the cloud-config, probably I will try to explore the other options and see if we can mask these details.